packages/application-extension/src/index.tsx

```diff
@@ -104,6 +104,9 @@ namespace CommandIDs {
 
   export const toggleSideTabBar: string = 'application:toggle-side-tabbar';
 
+  export const activateSidebarWidget: string =
+    'application:activate-sidebar-widget';
```

Comment:
```suggestion
  export const toggleSidebarWidget: string =
    'application:toggle-sidebar-widget';
```

For a better readability, we should rename the command (it should also be updated in *commands.json*).